packages/node_modules/@ciscospark/plugin-avatar/src/avatar-url-batcher.js

```diff
@@ -53,6 +53,9 @@ const AvatarUrlBatcher = Batcher.extend({
   },
 
   handleItemSuccess(item) {
+    const maxAge = item.response.cacheControl && item.response.cacheControl.indexOf(`max-age=`) !== -1 && Number(item.response.cacheControl.split(`max-age=`)[1]) || undefined;
+    item = {uuid: item.uuid, size: item.size, url: item.response.url, maxAge};
+
     return this.getDeferredForResponse(item)
```

Comment:
cacheControl value may have changed its format. Regardless, maxAge is now correctly parsed from the given cache control. Note plugin-avatar will ultimately throw an exception if cacheControl does not exist or fails the parse. This is to notify web devs as soon as possible if the protocol changes